docker/dockerfiles/Dockerfile_be_centos

```diff
@@ -0,0 +1,15 @@
+FROM centos:7
+
+ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en' LC_ALL='en_US.UTF-8'
+
+RUN yum install -y tzdata openssl curl vim ca-certificates fontconfig gzip tar mysql java-11-openjdk
+
+COPY ../../output/be /opt/starrocks/be/
+Copy ../bin/be_entrypoint.sh ../bin/be_prestop.sh /opt/starrocks/
```

Comment:
if docker build is running with `./dockerfiles` as build root, it is not possible to reference ../bin/ in dockerfile context

Reply:
please double check this file path.